aleph_scoring/metrics/models.py

```diff
@@ -27,6 +27,7 @@ class CcnMetrics(AlephNodeMetrics):
 class CrnMetrics(AlephNodeMetrics):
     diagnostic_vm_latency: Optional[float]
     full_check_latency: Optional[float]
+    vm_ping_response_time: Optional[float] = None
```

Comment:
Why not `vm_ping_latency` to keep the same syntax ? Or "Round Trip Time" to use the correct denomination ?